examples/window/low_power.rs

```diff
@@ -0,0 +1,69 @@
+use bevy::{
+    prelude::*,
+    window::RequestRedraw,
+    winit::{ControlFlow, WinitConfig},
+};
+
+/// This example illustrates how to run in low power mode, useful for making desktop applications.
+/// The app will only update when there is an event (resize, input, etc.), or you send a redraw
+/// request.
+fn main() {
+    App::new()
+        .insert_resource(WinitConfig {
+            control_flow: ControlFlow::Wait,
+            ..Default::default()
+        })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(rotate)
+        // Try uncommenting this system to manually request a redraw every frame:
+        //.add_system(request_redraw)
+        .run();
+}
+
+fn request_redraw(mut event: EventWriter<RequestRedraw>) {
+    event.send(RequestRedraw);
+}
+
+fn rotate(mut cube_transform: Query<&mut Transform, With<Rotator>>) {
+    for mut transform in cube_transform.iter_mut() {
+        transform.rotate(Quat::from_rotation_y(0.05));
+    }
+}
+
+#[derive(Component)]
+struct Rotator;
+
+/// set up a simple 3D scene
```

Comment:
Any specific suggestions? I didn't want to distract from what was important in the example.

Reply:
I was thinking either a) spawn like 10k cubes or b) do some fake work using sleep.

Reply:
What is the goal here, to show how much power usage drops?

Reply:
Yeah; we want to be able to let users actually verify that this works on their machine. Going from 1% draw to 0.1% will be much harder to spot.

Reply:
This is partly for convincing users that it works, but also to allow us to catch weird platform-specific bugs.

Reply:
I think just disabling vsync will do the trick, that will bog down CPU/GPU regardless of how powerful their PC is - it will basically just run as fast as it can.

Reply:
Perfect, that's nice and easy.

Reply:
Doesn't work on g-sync monitors. Mine will stay at 144hz even with v-sync off.

Reply:
> Doesn't work on g-sync monitors. Mine will stay at 144hz even with v-sync off.

Have you tested this? From what I understand, using immediate mode will prevent the main loop from waiting on sync, regardless of which frames are presented on screen - similar to mailbox mode.

In other words, you might see tear-free rendering capped at your monitor's refresh rate, but the bevy app will still be running as fast as it can in the background. I can only test on a freesync monitor.

Reply:
What I meant is that disabling vsync doesn't actually unlock the framerate on a g-sync monitor. At least, not consistently, if I move the window between a g-sync and non g-sync monitor it can sometimes actually unlock it. It's really weird, but @cwfitzgerald confirmed that g-sync is weird in this context with wgpu, My point is mostly that it won't change the cpu usage.

Reply:
https://discord.com/channels/691052431525675048/866787577687310356/927054585363853353 here's a link to a discord discussion about this

Reply:
I revived that conversation and there weren't any conclusive answers. Could you test the `low_power` example in this branch on the gsync monitor, and report back with resource usage between the three modes?